container/docker/factory.go

```diff
@@ -23,13 +23,13 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/fsouza/go-dockerclient"
 	"github.com/google/cadvisor/container"
 	"github.com/google/cadvisor/container/libcontainer"
 	"github.com/google/cadvisor/fs"
 	info "github.com/google/cadvisor/info/v1"
 	"github.com/google/cadvisor/utils"
 
-	"github.com/fsouza/go-dockerclient"
```

Comment:
How is this ordering guaranteed? If its through a special golang tool, that should be part of the build infra.